routers/routes/routes.go

```diff
@@ -353,10 +353,6 @@ func RegisterRoutes(m *macaron.Macaron) {
 		m.Group("", func() {
 			m.Get("/create", org.Create)
 			m.Post("/create", bindIgnErr(auth.CreateOrgForm{}), org.CreatePost)
-		}, func(ctx *context.Context) {
-			if !ctx.User.CanCreateOrganization() {
-				ctx.NotFound()
-			}
```

Comment:
Before anyone asks why this was removed: ctx.NotFound does not do what you expect it to - it does not return a 404 response. It registers the handlers for 404 responses, and in this case resets them. The logic for returning not found when someone tries to access /org/create has been moved to org.Create and org.CreatePost.